packages/@react-aria/interactions/src/useInteractOutside.ts

```diff
@@ -19,15 +19,17 @@ import {RefObject, SyntheticEvent, useEffect, useRef} from 'react';
 
 interface InteractOutsideProps {
   ref: RefObject<Element>,
-  onInteractOutside?: (e: SyntheticEvent) => void
+  onInteractOutside?: (e: SyntheticEvent) => void,
+  /** Whether the interact outside events should be disabled. */
+  isDisabled?: boolean
 }
 
 /**
  * Example, used in components like Dialogs and Popovers so they can close
  * when a user clicks outside them.
  */
 export function useInteractOutside(props: InteractOutsideProps) {
-  let {ref, onInteractOutside} = props;
+  let {ref, onInteractOutside, isDisabled} = props;
   let stateRef = useRef({
     isPointerDown: false,
```

Comment:
I think we should also add the same `if (isDisabled)` logic to the `onPointerDown` as well so we don't get stuck in `isPointerDown: true` upon clicking outside

Reply:
Fixed
4042f396